dynamic/wanted_attribute_builder.go

```diff
@@ -18,61 +17,46 @@ type WantedAttributeBuilder struct {
 
 // WantedAttribute represents a wanted attribute in a dynamic sharing policy
 type WantedAttribute struct {
-	name               string
-	derivation         string
-	constraints        []constraintInterface
-	acceptSelfAsserted bool
+	Name               string                `json:"name,omitempty"`
```

Comment:
I know it's just copied from the MarshallJSON method which we've now removed, but since we check that `Name` isn't null when building, I don't think it also makes sense to also `omitempty` here. Do you think we should remove this?

Reply:
Fixed in 32b9cf5